discovery-provider/src/utils/config.py

```diff
@@ -68,6 +68,9 @@ def parse_flask_section(self):
         if ('url_read_replica' not in current_app.config['db']) or (not current_app.config['db']['url_read_replica']):
             current_app.config['db']['url_read_replica'] = current_app.config['db']['url']
 
+        # Always disable (not included in app.default_config)
+        current_app.config['RESTX_MASK_SWAGGER'] = False
```

Comment:
@dmanjunath this cool with you? (dont see anything that could go wrong)

Reply:
yeah should be fine. could you just throw this link on there for reference? https://flask-restx.readthedocs.io/en/latest/mask.html#usage